docs/reference/components/sinks/humio_metrics.cue

```diff
@@ -0,0 +1,128 @@
+package metadata
+
+components: sinks: humio_metrics: {
+	title:             "Humio Metrics"
+	short_description: "Batches metrics events, converted to log entries, to [Humio][urls.humio] via the [HEC API][urls.humio_hec]."
+	long_description:  "[Humio][urls.humio] is a time-series logging and aggregation platform for unrestricted, comprehensive event analysis, On-Premises or in the Cloud. With 1TB/day of raw log ingest/node, in-memory stream processing, and live, shareable dashboards and alerts, you can instantly and in real-time explore, monitor, and visualize any system’s data. Metrics are converted to log events via the metric_to_log transform."
+
+	classes: {
+		commonly_used: false
+		function:      "transmit"
+		service_providers: ["Humio"]
+		egress_method: "batch"
+	}
+
+	features: {
+		batch: {
+			enabled:      true
+			common:       false
+			max_bytes:    1049000
+			max_events:   null
+			timeout_secs: 1
+		}
+		buffer: enabled: true
+		compression: {
+			enabled: true
+			default: null
+			gzip:    true
+		}
+		encoding: codec: {
+			enabled: true
+			default: "json"
+			enum: ["json", "text"]
+		}
+		healthcheck: enabled: true
+		request: {
+			enabled:                    true
+			in_flight_limit:            10
+			rate_limit_duration_secs:   1
+			rate_limit_num:             10
+			retry_initial_backoff_secs: 1
+			retry_max_duration_secs:    10
+			timeout_secs:               60
+		}
+		tls: enabled: false
+	}
+
+	statuses: {
+		delivery:    "at_least_once"
+		development: "beta"
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
```

Comment:
If these are exactly the same as `humio_logs` you could just do:

```
configuration: components.sinks.humio_logs.configuration
```

Reply:
And the same goes for most of the other settings here as well.

Reply:
Oh nice!

That will have to wait until #4359 gets merged.

Reply:
@FungusHumungus Just an FYI I merged #4359